README.md

```diff
@@ -49,7 +49,65 @@ devtools::install_github("UBC-MDS/feature-selection-r")
 
 ## Usage
 
-- TODO
+To guide you with an example of how to use this package, we would use the [Friedman dataset](https://www.rdocumentation.org/packages/tgp/versions/2.4-14/topics/friedman.1.data).
+
+Load dataset:
+```
+data <- dplyr::select(tgp::friedman.1.data(), -Ytrue)
+X <- data[1:(length(data)-1)]
+y <- data[length(data)]
```

Comment:
May I suggest slightly easier-to-read code?
```
X <- dplyr::select(data, -Y)
y <- dplyr::select(data, Y)
```

Reply:
I used also 'data' because in the recursive_feature_elimination used the complete data, but I just see the PR changing out to X and y. I will update this.